pkg/v1/providers/ytt/vendir/kapp-controller/_ytt_lib/addons/packages/kapp-controller/0.30.0/bundle/config/overlays/update-deployment.yaml

```diff
@@ -29,10 +29,42 @@ spec:
           #@overlay/match by="name"
           - name: KAPPCTRL_API_PORT
             value: #@ str(values.kappController.deployment.apiPort)
+        #@ if values.kappController.deployment.hostNetwork:
+        volumeMounts:
+          - mountPath: /etc
+            name: etc
+        #@ end
+      #@ if values.kappController.deployment.hostNetwork:
+      #! Using init container bypasses the restriction of not having root access in main container
+      #! It modifies /etc/resolv.conf which is shared by main container
+      initContainers:
+      - args:
+        - -c
+        - #@ generateBashCmdForDNS(values.kappController.deployment.serviceCIDR)
+        command:
+        - /bin/sh
+        #! Beware, update this image with each version bump!!!!!!!
```

Comment:
Moving the overlay to the package itself would remove the need to keeping in sync as kbld will be responsible for generating the entries.

Reply:
The t-f will use CRS to deploy CNI first, so we need this additional overlay to change the image.